### DISCUSSION TITLE
Add "Windows 11 Snap Layouts"

### DISCUSSION THREAD

**Discussion by marekmasek** [rxhanson/Rectangle#852] [Ideas]
Is it planned to add better mouse control, something like "Snap Layouts" in Windows 11?
Most of the time I have 3 windows with this layout: 1/4, 1/2, 1/4 and it's really slow to get this layout only with mouse in Rectangle.

![image](https://user-images.githubusercontent.com/48589322/178784609-ce1c3c8d-bb5f-43d1-8250-f0d21b1b4527.png)

In Windows 11 there are two options for that.
1. Hover the mouse over a maximize button and click on the layout. 
2. Drag a window to the top of the screen and drop it on top of the layout.

https://user-images.githubusercontent.com/48589322/178784668-63da048e-5454-446e-8f99-3ca6591bdd8d.mov

https://user-images.githubusercontent.com/48589322/178786715-e169dc07-a0c1-4702-9241-6b8e97f4085f.mov

**Comment by rxhanson** [comment]
Thanks for the feedback! I don't have plans to add it in to Rectangle at the moment, but see no reason to turn down good pull requests for anything like this.

I have had plans to add the second way to Rectangle Pro, prior to it being added in Windows 11, but I don't have any timeframe for when it would make it in. (Both of these options feel really slow compared to window throw in Rectangle Pro, but I get that some users want traditional style drag and drop / click solutions that can be done with one hand and a mouse).
